README.md

```diff
@@ -32,20 +32,30 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+The dom (Document Object Model) is a representative of the code and not the actual code. A tree like structure that represents the content structure and style. 
```

Comment:
The DOM is an API that creates a model of the window HTML (via the tree like structure you mention) which can then be manipulated with javascript. It serves as an intermediary between the window object and the javascript one would like to use to manipulate elements on the window object.